bin/bugsnag-maze-runner

```diff
@@ -75,12 +122,51 @@ class MazeRunnerEntry
     @args = args.dup
     options = @parser.parse args
 
+    # Process CL options
     init if options[:init]
-    MazeRunner.configuration.appium_session_isolation = true if options[OPTION_SEPARATE_SESSIONS]
+    MazeRunner.configuration.appium_session_isolation = options[OPTION_SEPARATE_SESSIONS]
+    MazeRunner.configuration.app_location = options[OPTION_APP]
+    farm = options[OPTION_FARM]
+    device_type = MazeRunner.configuration.device_type = options[OPTION_DEVICE]
+    MazeRunner.configuration.farm = if farm.nil?
+                                      :none
+                                    elsif farm == 'bs'
+                                      :bs
+                                    elsif farm == 'local'
+                                      :local
+                                    else
+                                      raise "Unknown farm '#{farm}'"
+                                    end
+
+    # Farm specific options
+    if MazeRunner.configuration.farm == :bs
+      raise "option --#{OPTION_USERNAME} must be specified" if options[OPTION_USERNAME].nil?
+      raise "option --#{OPTION_ACCESS_KEY} must be specified" if options[OPTION_ACCESS_KEY].nil?
+      raise "Device type '#{device}' not known on BrowserStack.  Must be one of #{Devices::DEVICE_HASH.keys}" \
+        unless Devices::DEVICE_HASH.keys.include? device_type
```

Comment:
Rather than wrap the line, I'd break it up for readability.
```suggestion
      unless Devices::DEVICE_HASH.keys.include? device_type
        raise "Device type '#{device}' not known on BrowserStack.  Must be one of #{Devices::DEVICE_HASH.keys}"
      end
```

Reply:
There's also [a `has_key?` method](https://ruby-doc.org/core-2.7.1/Hash.html#method-i-has_key-3F) you can use instead of `.keys.include?`

Reply:
Line separated.  I got used to Rubocop moaning that I should use favour the single line form and just did it, but apparently Rubocop is smart enough in this case to know that the increased line length would require wrapping.  

Rubocop has also told me to go further and just use `key?`.